DirectProgramming/DPC++FPGA/Tutorials/Tools/system_profiling/README.md

```diff
@@ -66,23 +66,23 @@ void profiling_example(const std::vector<float>& vec_in,
 
 This tutorial introduces the Intercept Layer for OpenCL* Applications, a profiling tool that extracts and visualizes system-level profiling information for DPC++ programs.  This tool can extract the same profiling data (and more) as the code snippet above, without requiring any code-level profiling directives.
 
-The Intercept Layer for OpenCL* provides coarse-grained, system-level profiling information. A complementary tool, the Intel® FPGA Dynamic Profiler for DPC++, provides fine-grained profiling information for the kernels executing on the device. Together, these two tools can be used to optimize both host and device side execution. However these tools should not be used at the same time, as the Intercept Layer for OpenCL* may slow down the runtime execution, rendering the Dynamic Profiler data less accurate. 
+The Intercept Layer for OpenCL* provides coarse-grained, system-level profiling information. A complementary tool, the Intel® FPGA Dynamic Profiler for DPC++, provides fine-grained profiling information for the kernels executing on the device. Together, these two tools can be used to optimize both host and device side execution. However, these tools should not be used simultaneously, as the Intercept Layer for OpenCL* may slow down the runtime execution, rendering the Dynamic Profiler data less accurate. 
 
 ### The Intercept Layer for OpenCL* Applications
 
 The Intercept Layer for OpenCL* Applications is an open-source tool that you can use to profile DPC++ designs at a system-level. Although it is not part of the oneAPI Base Toolkit installation, it is freely available on GitHub.
 
-This tool serves the following purpose:
-* Intercept host calls before they reach the device in order to gather performance data and log host calls.
-* Provide data to visualize the calls through time, and can separate them into *queued*, *submitted*, and *execution* sections for a better understanding of the execution.
-* Identify gaps (using visualization) in the runtime that may be leading to inefficient execution and throughput drops.
+This tool serves the following purposes:
+- Intercept host calls before they reach the device to gather performance data and log host calls.
+Provide data to visualize the calls through time and separate them into *queued*, *submitted*, and *execution* sections to understand the execution better.
```

Comment:
Random deletion of bullet -- also very frustrating among the many fixes I'm having to make.